internal/serialization/default_portable_reader.go

```diff
@@ -454,6 +465,23 @@ func (pr *DefaultPortableReader) runAtPosition(pos int32, f func()) {
 	pr.input.SetPosition(backup)
 }
 
+func (pr *DefaultPortableReader) readArrayOfTime(f func(input serialization.DataInput) time.Time) (ts []time.Time) {
+	l := pr.input.ReadInt32()
+	if l == nilArrayLength {
+		return
+	}
+	ts = make([]time.Time, l)
+	offset := pr.input.Position()
+	for i := int32(0); i < l; i++ {
+		//pr.input.SetPosition(offset + i*Int32SizeInBytes)
```

Comment:
@yuce this is also possible, I'm not sure about exposing `ObjectDataInput`. Wdyt?

Reply:
Let's keep it as is for now.

Reply:
Removed the comments describing the alternative 👍